spec/jobs/push_handover_dates_to_delius_job_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+RSpec.describe PushHandoverDatesToDeliusJob, type: :job do
+  let(:nomis_offender) { build(:nomis_offender) }
+  let(:offender_no) { nomis_offender.fetch(:offenderNo) }
+
+  before do
+    stub_auth_token
+    allow(HmppsApi::CommunityApi).to receive(:set_handover_dates)
```

Comment:
this is a very weak stub. IIMHO it should be a mock - expect().to receive(:set_handover_dates).with(<the correct parameters>)